.circleci/main.yml

```diff
@@ -2252,6 +2252,7 @@ commands:
     steps:
       - attach_workspace:
           at: ./
+      - run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev
```

Comment:
This dependency needs to be documented outside of CI, otherwise we can't run the extension locally